### PR TITLE
update README of gloo deployment

### PR DESCRIPTION
**Pull request by smnbbrv**
Hi guys,

this PR adds some adaption of the README to clarify two possible failure points.

P.S.: the change of the tables is unintentional (comes automatically from the formatter plugin for VSCode). If this is an issue, please let me know.

### CODE REVIEW

File: charts/dvpe-deployment-gloo/README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ This secret needs to contain the client_secret information as base64 encoded str
 Gloo expects a trailing newline at the secret string.
 You can encode your secret with the following command:
     ```bash
-    echo 'clientSecret: <clientSecretFromIDP>' | base64
+    echo -n 'clientSecret: <clientSecretFromIDP>' | base64
```
Comment (mithie) [inline charts/dvpe-deployment-gloo/README.md:64]: good point!

File: charts/dvpe-deployment-gloo/README.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ You can encode your secret with the following command:
       annotations:
         resource_kind: '*v1.Secret'
       name: webeam-oidc
-      namespace: argo-apps
+      namespace: <yourNamespace>
```
Comment (mithie) [inline charts/dvpe-deployment-gloo/README.md:77]: great, thanks!

File: charts/dvpe-deployment-gloo/README.md [hunk 3]
```diff
@@ -156,77 +156,77 @@ The following table lists the configurable parameters of the chart and its defau
...
-| service.spec.ports.http.port | string | `"80"` | The http port the service is exposed to in the cluster. |
-| service.spec.ports.http.targetPort | string | `"80"` | The http port the service listens to and to which requests will be sent. |
-| service.spec.ports.https.port | string | `"443"` | The https port the service is exposed to in the cluster. |
-| service.spec.ports.https.targetPort | string | `"443"` | The http port the service listens to and to which requests will be sent. |
-| service.spec.type | string | `"ClusterIP"` | Specify what kind of service to deploy. See [Kubernetes Service Spec](https://kubernetes.io/docs/concepts/services-networking/service/) for details |
+| Key                                                              | Type   | Default               | Description                                                                                                                                                                                                                                                    |
```
Comment (mithie) [inline charts/dvpe-deployment-gloo/README.md:159]: this table is generated via helm-docs. I would suggest that you execute helm-docs locally before you checkin. that's also the reason why the commit hooks weren't approved. 
you can have a look at https://github.com/norwoodj/helm-docs for details on how to use helm-docs

**Review by mithie — CHANGES_REQUESTED** [review]

File: charts/dvpe-deployment-gloo/README.md (hunk 1 above)
Comment (christodo) [reply charts/dvpe-deployment-gloo/README.md:64]: Gloo expectes a secret with a new line. `echo -n ` will not work

**Review by christodo — CHANGES_REQUESTED** [review]
see comment

**Comment by smnbbrv** [comment]
hi @christodo @mithie 

yep, sorry for delay.

The question is now whether gloo really expects the newline in the end of OIDC config. I still do not manage to get to the end of the quest with OIDC and validate that it really works from the beginning to the end.

Could you verify that it works for your team as well?

**Pull request closed without merging**